generate/input/descriptor.json

```diff
@@ -949,9 +949,11 @@
         "git_ignore_path_is_ignored": {
           "args": {
             "ignored": {
+              "shouldAlloc": true,
               "isReturn": true
             }
-          }
+          },
+          "isAsync": true
```

Comment:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  :tada: &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; :tada:
&nbsp;&nbsp;&nbsp;  :tada:  yeah, it's pretty ~cool~&nbsp; &nbsp; &nbsp; &nbsp;  :tada:
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  :tada: &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; :tada:
&nbsp;&nbsp;&nbsp;&nbsp; :tada:; &nbsp; &nbsp; :tada:
:penguin: